### ISSUE TITLE
definition of Discovery-enabled EPUB Publication

### ISSUE THREAD

**Issue by rdeltour** [w3c/epub-specs#907]
Section [2.1 Discovery-Enabled EPUB Publication](https://rawgit.com/IDPF/epub-revision/master/build/a11y/accessibility.html#sec-conf-discovery) reads:

> The following requirements apply to a discovery-enabled EPUB Publication

But _discovery-enabled EPUB Publication_ is never defined. Some suggestions, non-prioritzed:
- "discoverable" might sound better than "discovery-enabled" (with a grain of salt, coming from a non-native speaker)
- should we add "Discoverable EPUB Publication" to the terminology?
- for consistency with the following sections, I would reword the paragraph to "An EPUB Publication must meet the following requirements to be discoverable" or "A discoverable EPUB Publication must meet the following requirements" (with a link to the terminology.

**Comment by mattgarrish** [comment]
I recall using "discovery-enabled" because it's not the publication itself that achieves anything. In most case, it's going to be the distributor or reading system using the metadata to enable discovery. Someone could read the package document directly, but I don't imagine that'll be common.

Discoverable is easier to read, but it also sounds rather generic, like it could be about finding epubs wherever they've been hidden in the dark corners of the web. A definition would be needed.

I've adjusted the wording, but this might be one for a future call.
